tox.ini

```diff
@@ -47,8 +47,6 @@ commands =
 setenv = 
     DJANGO_SETTINGS_MODULE = enterprise_subsidy.settings.test
     PYTHONPATH = {toxinidir}
-    # Adding the option here instead of as a default in the docs Makefile because that Makefile is generated by shpinx.
-    SPHINXOPTS = -W
```

Comment:
I spent like 20 minutes trying to squash a sphinx warning, but couldn't figure it out before giving up.  This change just makes us ignore sphinx warnings.

Reply:
For future reference, here was the warning:

```
WARNING: autodoc: failed to import module 'urls' from module 'enterprise_subsidy'; the following exception was raised:
Traceback (most recent call last):
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/sphinx/ext/autodoc/importer.py", line 58, in import_module
    return importlib.import_module(modname)
  File "/home/tsankey/tmp/pyenv/versions/3.8.13/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/tsankey/tmp/edx/enterprise-subsidy/enterprise_subsidy/urls.py", line 20, in <module>
    from auth_backends.urls import oauth2_urlpatterns
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/auth_backends/urls.py", line 7, in <module>
    from auth_backends.views import (
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/auth_backends/views.py", line 10, in <module>
    from social_django.utils import load_strategy, load_backend
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/social_django/utils.py", line 15, in <module>
    Strategy = module_member(STRATEGY)
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/social_core/utils.py", line 58, in module_member
    module = import_module(mod)
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/social_core/utils.py", line 52, in import_module
    __import__(name)
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/auth_backends/strategies.py", line 7, in <module>
    from social_django.strategy import DjangoStrategy
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/social_django/strategy.py", line 4, in <module>
    from django.contrib.contenttypes.models import ContentType
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/django/contrib/contenttypes/models.py", line 133, in <module>
    class ContentType(models.Model):
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/django/db/models/base.py", line 108, in __new__
    app_config = apps.get_containing_app_config(module)
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/django/apps/registry.py", line 253, in get_containing_app_config
    self.check_apps_ready()
  File "/home/tsankey/tmp/edx/enterprise-subsidy/.tox/docs/lib/python3.8/site-packages/django/apps/registry.py", line 136, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
```